include/aspect/geometry_model/ellipsoidal_chunk.h

```diff
@@ -72,6 +73,8 @@ namespace aspect
             Point<3>
             push_forward(const Point<3> &chart_point) const;
 
+            const static InitialTopographyModel::Interface<dim> *topography;
```

Comment:
Is this the official way to use the `InitialTopographyModel`? We do not use static members much, but I was not involved in the initial design so maybe there is a reason that I do not know.